pkg/ebpf/tracer.go

```diff
@@ -113,6 +113,11 @@ func NewTracer(config *Config) (*Tracer, error) {
 		return nil, fmt.Errorf("could not load bpf module: %s", err)
 	}
 
+	// make sure debugfs is mounted
```

Comment:
You've changed the order of operations here - this should be the very first thing we check when creating a `Tracer`.